src/templates/cl_kg/cellxgene_marker_template.tsv

```diff
@@ -1,5 +1,5 @@
 ID	MARKER	MARKER_SCORE	CELL_RATIO	TISSUE	SPECIES
-ID	A hm:has_marker	>A hm:marker_score	>A hm:cell_ratio	>AI hm:context	>AI hm:species
+ID	A CLM:0009999	>A CLM:0009998	>A CLM:0010002	>AI CLM:0009997	>AI CLM:0009996
```

Comment:
Value of CLM:0009999 should be IRI `AI CLM:0009999`
Value of CLM:0009998 should be float `AT CLM:0009998^^xsd:integer`